src/lib/free-queue/free-queue.js

```diff
@@ -80,11 +77,11 @@ class FreeQueue {
     for (let i = 0; i < this._channelCount; ++i) {
       const startByteOffset = this._dataPtr + i * channelByteSize;
       const endByteOffset = startByteOffset + channelByteSize;
-      // Get the actual array index by dividing the byte offset by 2 bytes.
+      // Get the actual array index by dividing the byte offset by 4, the size of float. (right shift >> 2 === dividing by 4)
       this._channelData[i] =
           this._module.HEAPF32.subarray(
-              startByteOffset >> BYTES_PER_UNIT,
-              endByteOffset >> BYTES_PER_UNIT);
+              startByteOffset >> 2,
+              endByteOffset >> 2);
```

Comment:
For even more clarity, what do you think about either of these options?

1. Make the `2` a named constant like `FLOAT32_INDEX_SHIFT`

or

2. Use division by `BYTES_PER_SAMPLE` along with `Math.floor()`